BeVolt/BSP/Simulator/Src/BSP_PLL.c

```diff
@@ -1,12 +1,28 @@
 #include "BSP_PLL.h"
+#include <stdint.h>
+#include <stdlib.h>
+#include <sys/file.h>
+#include "simulator_conf.h"
+
+static char* file = GET_CSV_PATH(PLL_CSV_FILE);
 
 /**
  * @brief   Initialize the PLL so the system core frequency runs at your preferred frequency.
  * @param   None
  * @return  None
  */
 void BSP_PLL_Init(void) {
```

Comment:
This function should set the clock frequency to 80MHz. On the actual board, the clock is set to 16MHz automatically on startup, and this function is used to change the clock frequency to 80MHz